src/openforms/tests/test_x_forwarded_host.py

```diff
@@ -11,6 +11,7 @@ class XForwardedHostTests(SimpleTestCase):
 
     factory = RequestFactory()
 
+    @override_settings(USE_X_FORWARDED_HOST=False)
```

Comment:
this tests that our settings set it to disabled by default, the intent is coupling the test to the relevant setting and this change decouples it.

Reply:
I guess I popped off 1 to many of my stash. This was tripping my local test runs, from when I was using mitmproxy to test the Signicat DigiD interactions.

Reply:
could also be because of your `.env` configuration? commenting that out might help